src/client/containers/PickGamePage/PickGamePage.component.js

```diff
@@ -1,6 +1,26 @@
 import React from 'react';
 import HeaderTitle from '../../components/Title/Title';
+import GameDescription from '../../components/GameDescription/GameDescriptionComponent';
+import Footer from '../../components/FooterComponent/FooterComponent';
 
+import Modal from '../../components/Modal/ListModal';
+
+const toggleModal = () => {
+  document.querySelector('.modalbox').classList.toggle('showmodal');
+};
 export const PickGamePage = () => (
-  <HeaderTitle title="This is the Pick Game Page" />
+  <div>
+    <HeaderTitle title="Pick a Map" />
+    <section className="home bodycontainer">
+      <div className="bodywrapper map">
+        <GameDescription
+          title="Pick a Maps"
+          latitude="55.661869"
+          longitude="12.540742"
+          isOpen={() => <Modal onClick={() => toggleModal} />}
```

Comment:
What is going on there? are you passing a component as a function inside another component? 🤔  does that work?